sentry-android-core/src/main/AndroidManifest.xml

```diff
@@ -1,17 +1,22 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android">
-  <uses-permission android:name="android.permission.INTERNET"/>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools">
+
+  <uses-permission android:name="android.permission.INTERNET" />
+
+  <uses-sdk tools:overrideLibrary="androidx.compose.runtime" />
```

Comment:
hmm, curious why this is needed since we only have compileOnly deps for compose in this module?

Reply:
manifest merging started to fail, due to compose being min SDK 21, vs. us being min SDK 19